# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufLibWrapper.cpp
# Commit: 85974aa6ae0b
# Full Hash: 85974aa6ae0bf20be02a09ecfcd3aa6f27ebcb91
# Author: stransky <stransky@redhat.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 3
# Description:
#   Bug 1751987 [Linux] Implement VAAPIUtils to get basic VA-API info r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139299
# ==============================================================================

diff -r 469777a5cb87 -r 85974aa6ae0b widget/gtk/DMABufLibWrapper.cpp
--- a/widget/gtk/DMABufLibWrapper.cpp	Wed Feb 23 10:37:35 2022 +0000
+++ b/widget/gtk/DMABufLibWrapper.cpp	Wed Feb 23 11:47:06 2022 +0000
@@ -279,8 +279,8 @@
        "XRE_IsRDDProcess %d\n",
        gfx::gfxVars::UseEGL(), StaticPrefs::media_ffmpeg_vaapi_enabled(),
        gfx::gfxVars::CanUseHardwareVideoDecoding(), XRE_IsRDDProcess()));
-  return gfx::gfxVars::UseEGL() && StaticPrefs::media_ffmpeg_vaapi_enabled() &&
-         XRE_IsRDDProcess() && gfx::gfxVars::CanUseHardwareVideoDecoding();
+  return gfx::gfxVars::UseVAAPI() && XRE_IsRDDProcess() &&
+         gfx::gfxVars::CanUseHardwareVideoDecoding();
 }
 bool nsDMABufDevice::IsDMABufWebGLEnabled() {
   LOGDMABUF(